dem_stitcher/stitcher.py

```diff
@@ -251,6 +251,10 @@ def stitch_dem(bounds: list,
     # Used for filling in glo_30 missing tiles if needed
     stitcher_kwargs = locals()
 
+    # This variable is used later to determine if there is intersection
+    if fill_in_glo_30:
+        glo_90_missing_intersection = intersects_missing_glo_30_tiles(bounds)
```

Comment:
This doesn't fail because python evaluates only the boolean statements at the beginning of linked conditionals (sometimes called ["short-circuiting"](https://stackoverflow.com/a/2580142)). In this case, `glo_90_missing_intersection` never evaluates unless `fill_in_glo_30` is `True.

I think it would be clearer if I reassigned `fill_in_glo_30` using `and` with the intersection conditional, so I will do that.